test/e2e/csv_e2e_test.go

```diff
@@ -502,6 +502,13 @@ var _ = Describe("ClusterServiceVersion", func() {
 		})
 
 		It("remains in phase Succeeded when only one pod is available", func() {
+			Eventually(func() bool {
+				dep, err := c.GetDeployment(testNamespace, "deployment")
+				Expect(err).ShouldNot(HaveOccurred())
+				Expect(dep).ShouldNot(BeNil())
+				return dep.Status.ReadyReplicas == 2
+			}).Should(BeTrue())
```

Comment:
How do you feel about returning an integer instead and handling the comparison outside of the inline function block, e.g. `}).Should(Equal(2))`?

Reply:
It seems better.  I'll change it.  Thanks.